test/browser/lifecycles/shouldComponentUpdate.test.js

```diff
@@ -778,7 +778,7 @@ describe('Lifecycle methods', () => {
 		expect(scratch.innerHTML).to.equal('<p>bar</p>');
 	});
 
-	it('should reorder non-updating nested Fragment children', () => {
+	it.skip('should reorder non-updating nested Fragment children', () => {
```

Comment:
Temp skipped, wanted to view the benchmark diff

Reply:
The issue here is that we currently reorder correctly but only the first child. we have 3 children under every Fragment and the other two children are clobbered to the back